src/test/java/com/google/api/codegen/protoannotations/testdata/library_gapic.yaml

```diff
@@ -2,7 +2,7 @@ type: com.google.api.codegen.ConfigProto
 config_schema_version: 1.0.0
 language_settings:
   java:
-    package_name: com.google.gcloud.pubsub.v1
+    package_name: com.google.example.library.v1
```

Comment:
All changes in this file are just to make this file the same as the other library_gapic.yaml test resource that is used to generate old-style GAPICs.